packages/leadership-hero/main.js

```diff
@@ -0,0 +1,82 @@
+import xss from 'xss';
+import { basicHeroDataAdapter, matrixBasicHeroService } from '../../global/js/utils';
+import leadershipHeroTemplate from './leadership-hero.hbs';
+
+/**
+ * @module LeadershipHero
+ * @description A module for rendering the Leadership Hero component.
+ */
+export default {
+    /**
+     * Renders the Leadership Hero.
+     *
+     * @async
+     * @function main
+     * @param {Object} args - Configuration arguments for the listing.
+     * @param {Object} info - Contextual information including environment variables.
+     * @returns {Promise<string>} Rendered HTML from the Handlebars template.
+     */
+    async main(_args, info) {
+        const fnsCtx = info?.fns || info?.ctx || {};
```

Comment:
I've noticed that in case of getting current asset id we need to use only `ctx` object.
```suggestion
        const fnsCtx = info?.ctx || {};
```

Reply:
Changed, thanks!